napari_aicsimageio/core.py

```diff
@@ -7,13 +7,35 @@
 import xarray as xr
 from aicsimageio import AICSImage, exceptions, types
 from aicsimageio.dimensions import DimensionNames
+from qtpy.QtWidgets import QListWidget
+from qtpy.QtCore import Qt
+from napari import Viewer
 
 ###############################################################################
 
 LayerData = Union[Tuple[types.ArrayLike, Dict[str, Any], str]]
 PathLike = Union[str, List[str]]
 ReaderFunction = Callable[[PathLike], List[LayerData]]
 
+###############################################################################
+# _get_viewer() function from https://github.com/napari/napari/issues/2202
+# To provide access to the napari viewer to make the dock widget
+# Copyright (c) 2021 Jonas Windhager
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```

Comment:
If you want to keep these in just make them multiple lines please! Linter hates long lines :upside_down_face:

Reply:
If this is the MIT license or similar you may just want to remove all this text and say "Licensed under MIT License"

Reply:
I wasn't sure how to do this "properly"—this function is directly from @jwindhager. I think that including the full license—which I checked is compatible with the napari-AICSImageIO—is correct? 
I will definitely break up the lines though! Sorry!